README.md

```diff
@@ -156,6 +157,7 @@ From the KRACK <a href="https://www.krackattacks.com/">website</a>:
 | Sagemcom | No Known Official Response | N/A | 2017-10-17 | 2017-10-17 | |
 | Samsung Mobile | No Known Official Response | N/A | 2017-10-16 | 2017-10-16 | 28 Aug 2017 |
 | Sharp | No Known Official Response | N/A | 2017-10-16 | 2017-10-16 |  |
+| SnapAV | Araknis Networks - Wireless Access Points, Autonomic - 1e Music Streamer | "We are currently working on a firmware update, which will be available on SnapAV’s website, as well as OvrC." | 2017-10-16 | 2017-10-17 |  |
```

Comment:
Almost good to merge -- If there is no official URL for SnapAV, please mark that column as "No Known Official Response"

Thanks!